projects/extension/sql/idempotent/001-openai.sql

```diff
@@ -3,12 +3,12 @@
 -- openai_tokenize
 -- encode text as tokens for a given model
 -- https://github.com/openai/tiktoken/blob/main/README.md
-create or replace function ai.openai_tokenize(model text, text_input text) returns int[]
+create or replace function ai.openai_tokenize(_model text, _text text) returns int[]
```

Comment:
we do not use leading underscores in parameter names. They make it hard to call functions with `=>` syntax.  Even if it makes the developers job easier to use the underscore, it makes the user interface worse, and that's not a good tradeoff. This needs to be fixed all over.

Reply:
Got it, will make the change. This was an artifact from the fact that I was already building these functions prior to finding your project, and copy/paste.